src/Microsoft.Sbom.Extensions.DependencyInjection/Microsoft.Sbom.Extensions.DependencyInjection.csproj

```diff
@@ -18,6 +18,7 @@
 
   <ItemGroup>
     <PackageReference Include="AutoMapper.Extensions.Microsoft.DependencyInjection" />
+    <PackageReference Include="Microsoft.Build.Utilities.Core" />
```

Comment:
I really don't like the idea of adding the `Microsoft.Build.*` packages to the standalone tool. They were included exclusively for the functionality provided by the `Microsoft.Sbom.Targets` project, and I'd love to keep things that way. Is there a way to move the logging mapping out of the dependency injection assembly? One thought that comes to mind is to replace the `TaskLoggingHelper` parameter with some sort of delegate, and placing the creation of the `MSBuildLogger` within that delegate. If no delegate is provided, then the existing code gets run